internal/python/plan.go

```diff
@@ -318,7 +318,7 @@ func determineInstallCmd(ctx *pythonPlanContext) string {
 }
 
 func determineAptDependencies(ctx *pythonPlanContext) []string {
-	deps := []string{"build-essential"}
+	deps := []string{"build-essential", "nginx"}
```

Comment:
Is this dependency necessary in every Python applications?

Reply:
no, only wsgi ,i will change